tests/utils.py

```diff
@@ -839,9 +839,10 @@ def both_dirs_ready(n, scid):
         wait_for(lambda: both_dirs_ready(nodes[-1], scids[0]))
 
         # Make sure we have all node announcements, too (just check ends)
-        for n in nodes:
-            for end in (nodes[0], nodes[-1]):
-                wait_for(lambda: 'alias' in only_one(end.rpc.listnodes(n.info['id'])['nodes']))
+        if not privatechannels:
```

Comment:
Hmm announce=True and privatechannels=True is an invalid combination (which kinda reinforces my preference to call them 'wait_for_announce' and 'announce' BTW)  Just assert() above that they're not asking for it?

Reply:
Great point. Updated.